requirements.txt

```diff
@@ -1,30 +1,30 @@
 # pythonwhat deps
```

Comment:
By working on dependencies for workspace, I noticed that tensorflow sets quite strict dependency requirements. It shouldn't matter much for this PR, but in the base image / shared image / course you might need different versions of e.g. numpy (depending on how you lock versions, there might be conflicts with these versions).

Reply:
Good tip, Thank you, downgraded numpy and h5py to what the latest tensorflow uses.